### PR TITLE
Patch for Arcconf plugin

### PR DESCRIPTION
**Pull request by sharath-t**
Fix for below commands/APIs
arcconf:// list --type systems
arcconf:// list --type volumes
arcconf:// list --type pools
arcconf:// list --type disks
arcconf:// volume-delete --vol <id>
arcconf:// volume-raid-create --name <name> --disk <id> --raid-type <RAID TYPE>

support for below commands/APIs
cmd: lsmcli -u arcconf:// volume-raid-info --vol
cmd: lsmcli -u arcconf:// pool-member-info --pool
cmd: lsmcli -u arcconf:// volume-enable --vol
cmd: lsmcli -u arcconf:// volume-ident-led-on --vol
cmd: lsmcli -u arcconf:// volume-ident-led-off --vol
cmd: lsmcli -u arcconf:// list --type batteries

CI Kick

### CODE REVIEW

File: plugin/arcconf/arcconf.py [hunk 1]
```diff
@@ -41,17 +42,18 @@
 DRIVE_STATE_UNKNOWN = 20
 
 # Controller states
-CONTROLLER_STATUS_WORKING = 0
+CONTROLLER_STATUS_WORKING, CONTROLLER_STATUS_WORKING_DISPLAY = 0, 'Optimal'
```
Comment (cathay4t) [inline plugin/arcconf/arcconf.py]: Can this(and many other lines) split into two lines?

Not a bid deal, just to align with other codes.

File: plugin/arcconf/arcconf.py [hunk 2]
```diff
@@ -291,6 +314,7 @@ def _arcconf_exec(self, arcconf_cmds, flag_force=False):
         if flag_force:
             arcconf_cmds.append('noprompt')
         try:
+            #  print(arcconf_cmds)
```
Comment (cathay4t) [inline plugin/arcconf/arcconf.py]: Please remove this line.

**Comment by sharath-t** [comment]
@cathay4t , i have filed new PRs from #359 to #369 . 
you may close this PR.

**Comment by tasleson** [comment]
@sharath-t Would you be willing to run a CI test node on a system that has the hardware so we can automate testing of this plugin?

**Comment by sharath-t** [comment]
sure, i will try to set it up and give it a go in couple of days.

**Comment by sharath-t** [comment]
Attaching output file for 'sudo ./test/plugin_test.py --uri 'arcconf://''
[lsmcli_arcconf_test.txt](https://github.com/libstorage/libstoragemgmt/files/2995272/lsmcli_arcconf_test.txt)

**Comment by sharath-t** [comment]
> @sharath-t Would you be willing to run a CI test node on a system that has the hardware so we can automate testing of this plugin?
> 
> The code is found here: https://github.com/tasleson/lsm-ci

can you let me know what the code exactly do? my server in behind a private network.

**Comment by tasleson** [comment]
> > @sharath-t Would you be willing to run a CI test node on a system that has the hardware so we can automate testing of this plugin?
> > The code is found here: https://github.com/tasleson/lsm-ci
> 
> can you let me know what the code exactly do? my server in behind a private network.

The CI node works by connecting to a central node manager, thus it will work on private networks with no incoming network ports.  I'll send you an email with additional information which will include some SSL certificates to allow you to connect.

**Pull request closed without merging**